### PR TITLE
Update log messages

### PR DESCRIPTION
**Pull request by varsha888**
https://github.com/mobilecoinfoundation/mobilecoin/issues/2405
=> Changed log levels
=> Changed log messages
=> Set get block contents max sleep time to 60 seconds

### CODE REVIEW

File: watcher/src/watcher_db.rs [hunk 1]
```diff
@@ -456,7 +456,11 @@ impl WatcherDB {
                 Ok((ts, res)) => match res {
                     TimestampResultCode::WatcherBehind => {
                         if watcher_behind_timer.elapsed() > watcher_timeout {
-                            log::warn!(self.logger, "watcher is still behind on block index = {} after waiting {} seconds, caller will be blocked", block_index, watcher_timeout.as_secs());
+                            log::warn!(
+                                self.logger,
+                                "Caller will be blocked in {} seconds",
```
Comment (jgreat) [inline watcher/src/watcher_db.rs:461]: Do we lose some context there about why "Caller will be blocked?"  I think the original message seems to have more human context.

**Review by jgreat — COMMENTED** [review]
I don't see a change for the "Archive block retrieved for Url" message?  Is that going to be in a different PR?

File: ledger/sync/src/ledger_sync/ledger_sync_service.rs [hunk 2]
```diff
@@ -719,7 +721,12 @@ fn get_block_contents<TF: TransactionsFetcher + 'static>(
                                     // Sleep, with a linearly increasing delay. This prevents
                                     // endless retries
                                     // as long as the deadline is not exceeded.
-                                    thread::sleep(Duration::from_secs(num_attempts + 1));
+                                    let interval = if num_attempts + 1 < MAX_SLEEP_INTERVAL {
+                                        num_attempts + 1
+                                    } else {
+                                        MAX_SLEEP_INTERVAL
+                                    };
+                                    thread::sleep(Duration::from_secs(interval));
```
Comment (samdealy) [inline ledger/sync/src/ledger_sync/ledger_sync_service.rs:729]:
```suggestion
let next_interval = Duration::secs_from(num_attempts + 1);
let interval = std::cmp::min(next_interval, &MAX_SLEEP_INTERVAL)
thread::sleep(interval);
```

**Review by samdealy — CHANGES_REQUESTED** [review]

File: watcher/src/watcher_db.rs (hunk 1 above)
Comment (varsha888) [reply watcher/src/watcher_db.rs:461]: I'll change it back

**Review by jcape — APPROVED** [review]

**Pull request merged**